week-1/H-numbers/exercise.js

```diff
@@ -1 +1,7 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var numberOfStudents = 15;
+var numberOfMentors = 8;
+console.log(
+  "Total number of students and mentors: " +
```

Comment:
excellent, good use of parentheses to make it clear also what the code is doing. Nice!